### ISSUE TITLE
Can i change the crypto for bank money or items to win ?

### ISSUE THREAD

**Issue by parkzinc** [censorfriendly/esx-racing#2]
I need use bank money or give items to my users for win money or use in another things, can you help me to do this ?

**Comment by censorfriendly** [comment]
I haven’t made any updates to this script in a while. It should be easily modifiable to give cash or items. Do you have this script installed on your server and working?

**Comment by parkzinc** [comment]
Yes! your script is **AMAZING** man, perfect for my project, im doing a esx racing gamemode. changing sql lines to column "credits" i can use this credits with other scripts (core_credits) 

example for show  :     
```
MySQL.Async.fetchAll('Select * from users WHERE identifier = @identifier', {['@identifier'] = identifier}, function(result)
        TriggerClientEvent('racing:setCrypto', usource, result[1].credits)
```

i added a set in F6 to open the tablet and works sweet!
im trying to freeze players in the start position when countdown starts now.

Thank you very much for your work :)

**Comment by censorfriendly** [comment]
Ive added my discord to my profile, you can feel free to message me there. 
But as far as displaying those core credits what you have there definitely will work.
You would need to modify the finished race function to add credits to the players. Should be able to modify the column to point to the one you currently use and it should work just as it was intended
